base/boot.jl

```diff
@@ -824,9 +824,13 @@ Integer(x::Union{Float16, Float32, Float64}) = Int(x)
 # `_parse` must return an `svec` containing an `Expr` and the new offset as an
 # `Int`.
 #
-# The internal jl_parse which will call into Core._parse if not `nothing`.
+# The internal jl_parse will call into Core._parse if not `nothing`.
 _parse = nothing
 
+function set_parser(parser)
+    global _parse = parser
+end
```

Comment:
```suggestion
_setparser!(parser) = setglobal!(@__MODULE__, :_parse, parser)
```
is a bit simpler (and also make it more explicit that this is very internal function)?

Reply:
oh cool `setglobal!` is new to me.  Seems good.